pages/linux/trash.md

```diff
@@ -0,0 +1,28 @@
+# trash
+
+> A CLI for managing your trashcan / recycling bin.
+> More information: <https://github.com/andreafrancia/trash-cli>.
+
+- Delete a file (send to trash):
+
+`trash {{path/to/file}}`
+
+or
```

Comment:
I think this is messing up the linter. Perhaps you could remove this?

While the `trash-put` might do the same thing, I'd probably suggest that it has it's own page, as generally speaking each command will have it's own page.

Reply:
fixed